src/metabase/sync/util.clj

```diff
@@ -290,7 +290,7 @@
 (defn db->sync-tables
   "Return all the Tables that should go through the sync processes for `database-or-id`."
   [database-or-id]
-  (t2/select Table, :db_id (u/the-id database-or-id), :active true, :visibility_type nil))
+  (t2/select Table, :db_id (u/the-id database-or-id), :visibility_type nil))
```

Comment:
why?

Reply:
My bad, that wasn't meant to be committed